<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_scanarea">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="title">
      <string>2D Scan Area</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Step Size</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="hspan_doubleSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="h1_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="v1_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="h0_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="scan_step_comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="h_label">
        <property name="text">
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="v_label">
        <property name="text">
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="vspan_doubleSpinBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Dim. (H x V)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="v0_doubleSpinBox">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="center_pushButton">
        <property name="text">
         <string>Center on Pos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_stageposition">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Current Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" rowspan="2" colspan="5">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>xy</string>
        </property>
        <widget class="QPushButton" name="x_down_pushButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="y_up_pushButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>^</string>
         </property>
        </widget>
        <widget class="QPushButton" name="x_up_pushButton">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="y_down_pushButton">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>v</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>27</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>27</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="x_pos_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="y_pos_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="xy_step_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>20</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="xy_step_comboBox">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>20</y>
           <width>47</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>xy step:</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="7" column="1" rowspan="2" colspan="5">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>z</string>
        </property>
        <widget class="QPushButton" name="z_up_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="z_pos_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="z_down_pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>39</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="z_step_comboBox">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>50</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="z_step_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>47</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>z step:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stop_stage_pushButton">
         <property name="geometry">
          <rect>
           <x>244</x>
           <y>20</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop Stage Motors</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_scancontrol">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>Ocean Optics Spectrometer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="6" column="0">
       <widget class="QPushButton" name="oo_spec_set_bg_pushButton">
        <property name="text">
         <string>Set background</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="oo_spec_save_pushButton">
        <property name="text">
         <string>Save h5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="oo_spec_startstop_pushButton">
        <property name="text">
         <string>Start/Stop</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="oo_spec_bgsub_checkBox">
        <property name="text">
         <string>BG Subtract</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="oo_spec_acquire_cont_checkBox">
        <property name="text">
         <string>Acquire Continuously</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="oo_spec_int_time_doubleSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Ocean Optics Optimizer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="2">
       <widget class="QCheckBox" name="power_meter_acquire_cont_checkBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Acquire
Continuously</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="power_meter_power_label">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0.000 W</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_flircam">
     <property name="title">
      <string>Flircam</string>
     </property>
     <widget class="QGroupBox" name="groupBox_image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>361</width>
        <height>251</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>251</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>361</width>
        <height>361</height>
       </size>
      </property>
      <property name="title">
       <string>Image</string>
      </property>
      <widget class="QPushButton" name="flircam_show_pushButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_control">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>361</width>
        <height>50</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <widget class="QDoubleSpinBox" name="flircam_int_time_doubleSpinBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Exp. t</string>
       </property>
      </widget>
      <widget class="QComboBox" name="flircam_auto_exp_comboBox">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto Exp:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="flircam_live_checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>70</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Live</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="flircam_autolevel_checkBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Autolvl</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
